examples/protocol_specification_ex/output/two_party_negotiation/message.py

```diff
@@ -1,18 +1,33 @@
 """This module contains two_party_negotiation's message definition."""
 
 from enum import Enum
-from typing import Set, Tuple, cast
+from typing import Dict, FrozenSet, Optional, Set, Tuple, Union, cast
 
 from aea.protocols.base import Message
 
 DEFAULT_BODY_SIZE = 4
 
 
+class Unit:
+    """This class represents an instance of Unit."""
+
+    def __init__(self):
+        """Initialise an instance of Unit."""
+        raise NotImplementedError
+
+    def __eq__(self, other):
+        """Compare two Unit instances."""
+        if type(other) is type(self):
```

Comment:
I'd drop the if, else and just have `raise NotImplementedError`.

Reply:
Keep just __init__

Reply:
Done